docs/pages/machine_learning.rst

```diff
@@ -86,6 +86,59 @@ query can be given). The model can than be used in subsequent calls to ``PREDICT
 using the given name.
 Have a look into :ref:`ml` for more information.
 
+4. Check Model parameters - Model meta data
+-------------------------------------------
+After the Model was trained, you can inspect and get model details by using following
+sql statements
+
+.. code-block:: sql
+
+    -- show list of models  which are trained and saved in the context.
+    SHOW MODELS
+
+    -- To get Hyperparameters of the trained MODEL use ,
+    -- DESCRIBE MODEL <model_name>.
+    DESCRIBE MODEL my_model
+
+
+5. Export Trained Model
+------------------------
+Once your model was trained and performs good in your validation dataset,
+you can export the model into a file with supporting model serilization
+format like pickle,joblib,mlflow (framework agnostic serilization format) etc.
+
+Currently dask_sql supports pickle,joblib and mlflow
+formats for exporting the trained model which can be deployed as microservices etc
+
+Before training and exporting the models from different framework like
+lightgbm, catboost, please ensure the relevant packages are installed in the
+dask_sql environment otherwise it will raises RuntimeError and if you
+are using mlflow as model_serde ensure mlflow was installed
```

Comment:
If you change the name of the parameter, also change it here :-)

Reply:
updated 👍